init.lua

```diff
@@ -2,9 +2,13 @@
 -- ===================
 -- See LICENSE.txt for licensing and README.md for other information.
 
+-- internationalization boilerplate
+local MP = minetest.get_modpath(minetest.get_current_modname())
```

Comment:
better call it `modpath` (MP stands generally for ModPack and var names shouldn't have capital letters here)